examples/loftq_finetuning/quantize_save_load.py

```diff
@@ -184,54 +183,8 @@ def quantize_and_save():
     return base_model_dir, lora_model_dir
 
 
-def load_loftq(base_model_path, lora_adapter_path):
```

Comment:
Why this has been removed?

Reply:
I think this function was supposed to confirm that everything works fine after the loftq weight initialization step. Hence, not a required step.

Reply:
Perfect!